lib/node_modules/@stdlib/stats/base/dvarmpn/include/stdlib/stats/base/dvarmpn.h

```diff
@@ -31,7 +31,11 @@ extern "C" {
 /**
 * Computes the variance of a double-precision floating-point strided array provided a known mean and using Neely's correction algorithm.
 */
-double stdlib_strided_dvarmpn( const int64_t N, const double mean, const double correction, const double *X, const int64_t stride );
+double API_SUFFIX( stdlib_strided_dvarmpn )( const CBLAS_INT N, const double mean, const double correction, const double *X, const CBLAS_INT strideX );
+/**
+* Computes the variance of a double-precision floating-point strided array provided a known mean and using Neely's correction algorithm using alternative indexing semantics.
+*/
+double API_SUFFIX( stdlib_strided_dvarmpn_ndarray )( const CBLAS_INT N, const double mean, const double correction, const double *X, const CBLAS_INT strideX, const CBLAS_INT offsetX );
```

Comment:
```suggestion
double API_SUFFIX(stdlib_strided_dvarmpn_ndarray)( const CBLAS_INT N, const double mean, const double correction, const double *X, const CBLAS_INT strideX, const CBLAS_INT offsetX );
```
we don't give spacing in brackets when wrapping in `API_SUFFIX`

Reply:
kindly check all other instances as well incase i miss any!

Reply:
Resolved